### PR TITLE
Update and rename LICENSE.txt to LICENSE

### PR DESCRIPTION
**Pull request by vyaspranjal33**
Updated Year 2017 -> 2018

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 Nexmo
+Copyright (c) 2018 Nexmo
```
Comment (tbedford) [inline LICENSE]: Can we change to a range and specify Nexmo is a limited liability company:

Copyright (c) 2017-2018 Nexmo Ltd.

**Comment by vyaspranjal33** [comment]
@tbedford  changes have been made

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 Nexmo
+Copyright (c) 2017-2018 Nexmo Ltd .
```
Comment (tbedford) [inline LICENSE]: Sorry please remove space after Ltd. So it should be "Ltd." Thanks!

Comment (vyaspranjal33) [reply LICENSE]: done @tbedford

**Pull request merged**